src/index.ts

```diff
@@ -0,0 +1,97 @@
+import express, { Request, Response } from 'express';
+import swaggerJsdoc, { OAS3Definition } from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+import { Application } from 'express';
+
+interface ServerSatatus {
+  status: string;
+  message: string;
+}
+
+const app: Application = express();
+const port = 3000;
+const serverIsWorking: ServerSatatus = {
+  status: 'ok',
+  message: 'Server is working!',
+};
+
+const swaggerOptions: OAS3Definition = {
+  swaggerDefinition: {
+    openapi: '3.0.0',
+    info: {
+      title: 'My API Documentation',
+      version: '1.0.0',
+      description: 'Documentation for my API',
+    },
+    servers: [
+      {
+        url: `http://localhost:${port}`,
+        description: 'Local server',
+      },
+    ],
+  },
+  apis: ['src/index.ts'],
+  openapi: '',
+  info: {
+    title: 'test swagger',
+    version: '0.0.1',
+  },
+};
+
+const swaggerSpec: object = swaggerJsdoc(swaggerOptions);
```

Comment:
As `index.ts` file grows, may be better to start splitting things and move to it own separate modules. For example, swagger related configs can be move to `swagger.ts` module and then imported here.

Reply:
done